apps/web/components/Forms/PoolForm.tsx

```diff
@@ -377,65 +408,69 @@ export default function PoolForm({ alloAddr, token, communityAddr }: Props) {
                 ),
               )}
             </div>
-            <div className="mb-6 flex flex-col">
-              <div className="flex max-w-64 flex-col">
-                <FormInput
-                  label="Spending limit"
-                  register={register}
-                  required
-                  errors={errors}
-                  registerKey="spendingLimit"
-                  type="number"
-                  placeholder="20"
-                  readOnly={optionType !== 0}
-                  className="pr-14"
-                  otherProps={{
-                    step: 1 / PERCENTAGE_PRECISION,
-                    min: 1 / PERCENTAGE_PRECISION,
```

Comment:
variable name PERCENTAGE_PRECISION looks wrong here, the value its 10**7 it means that its the same as the D varible in the contracts, its PRECISION/SCALE number but not related to percentages numbers.

Fow now we can keep it, but we should refactor it later on